man/man8/zpool-scrub.8

```diff
@@ -92,9 +93,26 @@ Once resumed the scrub will pick up from the place where it was last
 checkpointed to disk.
 To resume a paused scrub issue
 .Nm zpool Cm scrub
+or
+.Nm zpool Cm scrub
+.Fl e
 again.
 .It Fl w
 Wait until scrub has completed before returning.
+.It Fl e
+Only scrub blocks in the error log.
+It is strongly advised to run error scrubs with keys loaded and filesystems
+mounted in the case of encrypted filesystems.
+Otherwise, errors in encrypted filesystems that are unmounted or have unloaded
+keys will be reported as belonging to the root filesystem.
```

Comment:
Of course you are correct!! I just realized I should have used `dsl_dataset_hold_obj_flags()` instead of `dsl_dataset_hold_obj()` all along in `head_errlog` (like in `check_filesystem()`) to have access to encrypted filesystems without being mounted. I will fix this.

Reply:
I just opened #14837.